src/schema/artwork/index.ts

```diff
@@ -560,11 +560,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
             return "Free shipping worldwide"
           var domesticShipping = amount(
-            ({ domestic_shipping_fee_cents }) => domestic_shipping_fee_cents
+            ({ domestic_shipping_fee_cents }) =>
+              domestic_shipping_fee_cents || null
```

Comment:
Why do we need the added `|| null` here? Don't we already return `null` from the money helper if this is undefined?

Reply:
In case domestic_shipping_free_cents is 0, to preserve the old behaviour.